src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -67,15 +67,15 @@ public partial class AccessViolationException : System.SystemException
     public delegate void Action<in T1, in T2, in T3, in T4, in T5, in T6, in T7, in T8, in T9>(T1 arg1, T2 arg2, T3 arg3, T4 arg4, T5 arg5, T6 arg6, T7 arg7, T8 arg8, T9 arg9);
     public static partial class Activator
     {
+        public static System.Runtime.Remoting.ObjectHandle CreateInstance(string assemblyName, string typeName) { throw null; }
```

Comment:
From https://github.com/dotnet/corefx/pull/30809

cc @MarcoRossignoli, please review for correctness

Reply:
too late for me...but seems ok...one curiosity, it works also for new ref project?I mean if we create an empty file and run tools...for issue like https://github.com/dotnet/corefx/issues/33117

Reply:
Good question!

If the file doesn't exist, then the tool throws an exception (can't find the expected file). We can probably fix it to create the ref file or at least fail gracefully.

```text
Unhandled Exception: System.IO.DirectoryNotFoundException: Could not find a part of the path 'E:\GitHub\Fork\corefx\src\System.Json\ref\System.Json.cs'.
     at System.IO.FileStream.ValidateFileHandle(SafeFileHandle fileHandle)
     at System.IO.FileStream.CreateFileOpenHandle(FileMode mode, FileShare share, FileOptions options)
     at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options)
     at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding, Int32 bufferSize)
     at System.IO.StreamWriter..ctor(String path, Boolean append)
     at System.IO.File.CreateText(String path)
     at GenAPI.Program.Main(String[] args)
```
cc @weshaggard 

If, however, we create an empty file first in the correct folder (ref directory), **then yes, it would work.**

I just tried it with System.Json and it worked great. We just have to make sure to add back Obsolete or EditorBrowsableNever attributes that might exist in the source.

cc @danmosemsft